src/app/cats/cats-overview/cats-overview.component.html

```diff
@@ -1,6 +1,18 @@
+
 <section>
-    <div class="listElement" *ngFor="let cat of cats">
-        <img src="./assets/cats-pic/{{cat.picUrl}}" alt="" (click)="onClick(cat)">
-        <span>{{cat.name}}</span>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <div class="sort-buttons"> 
+        <button  (click)="sortByName()">Alphabetical order</button>
+        <button (click)="sortByBreed()">Cat breed order</button>  
+    </div>
+  
+    <div [ngClass]="{'red-border' : cat.reserved == true}" class="listElement" *ngFor="let cat of cats" >
```

Comment:
nawet nie potrzeba porównania, to tak jak java czy C#, `if(cat.reserved)` to to samo co `if(cat.reserved === true)`